src/statd/python/yanger/yanger.py

```diff
@@ -587,7 +594,6 @@ def get_brport_multicast(ifname):
 
     return multicast
 
-
```

Comment:
Isn't double-newlines some PEP8 thing?

Reply:
Indeed it is, between methods/classes/etc. outside of a class, iirc.